MinimedKit/PumpEvents/DailyTotal523PumpEvent.swift

```diff
@@ -33,7 +33,7 @@ public struct Sara6EPumpEvent: PumpEvent {
     
     public var dictionaryRepresentation: [String: AnyObject] {
         return [
-            "_type": "Sara6E",
+            "_type": "DailyTotal523",
```

Comment:
Yeah, I did some searching through the CSVs and I don't see anything that looks like the daily summary either.